src/picongpu/include/plugins/PluginController.hpp

```diff
@@ -166,7 +165,8 @@ class PluginController : public ILightweightPlugin
         EnergyParticles<bmpl::_1>,
         BinEnergyParticles<bmpl::_1>,
         LiveViewPlugin<bmpl::_1>,
-        PositionsParticles<bmpl::_1>
+        PositionsParticles<bmpl::_1>,
+        ChargeConservation<bmpl::_1>
```

Comment:
pls use a Foreach for each species and use it with a sum of charges on FieldTmp :)